### ISSUE TITLE
`the-well-maintained-test` `v0.5.0` is unable to find `questions.json`

### ISSUE THREAD

**Issue by matthewfeickert** [ryancheley/the-well-maintained-test#14]
**Describe the bug**

Between `v0.4.0` and `v0.5.0` there was some change that has resulted in `the-well-maintained-test` not being able to find `questions.json` and raising 

```
FileNotFoundError: [Errno 2] No such file or directory: 'questions.json'
```

for both the `questions` and `url` commands.

**To Reproduce**
1. Install `the-well-maintained-test` `v0.5.0` from PyPI
2. Run `the-well-maintained-test questions` or `the-well-maintained-test url <url>` at the CLI

```console
$ docker run --rm -ti python:3.9 /bin/bash
root@fabc82d7adc8:/# python -m venv venv && . venv/bin/activate
(venv) root@fabc82d7adc8:/# python -m pip --quiet install --upgrade pip setuptools wheel
(venv) root@fabc82d7adc8:/# python -m pip --quiet install 'the-well-maintained-test==0.5.0'
(venv) root@fabc82d7adc8:/# python -m pip list
Package                  Version
------------------------ ---------
certifi                  2021.10.8
charset-normalizer       2.0.8
click                    8.0.3
colorama                 0.4.4
commonmark               0.9.1
idna                     3.3
pip                      21.3.1
Pygments                 2.10.0
requests                 2.26.0
rich                     10.15.0
setuptools               59.4.0
the-well-maintained-test 0.5.0
urllib3                  1.26.7
wheel                    0.37.0
(venv) root@fabc82d7adc8:/# the-well-maintained-test questions
Traceback (most recent call last):
  File "/venv/bin/the-well-maintained-test", line 8, in <module>
    sys.exit(cli())
  File "/venv/lib/python3.9/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/venv/lib/python3.9/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/venv/lib/python3.9/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/venv/lib/python3.9/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/venv/lib/python3.9/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/venv/lib/python3.9/site-packages/the_well_maintained_test/cli.py", line 167, in questions
    with open("questions.json") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'questions.json'
(venv) root@fabc82d7adc8:/# python -m pip --quiet install 'the-well-maintained-test==0.4.0'
(venv) root@fabc82d7adc8:/# the-well-maintained-test questions
1. Is it described as “production ready”?
2. Is there sufficient documentation?
3. Is there a changelog?
4. Is someone responding to bug reports?
5. Are there sufficient tests?
6. Are the tests running with the latest <Language> version?
7. Are the tests running with the latest <Integration> version?
8. Is there a Continuous Integration (CI) configuration?
9. Is the CI passing?
10. Does it seem relatively well used?
11. Has there been a commit in the last year?
12. Has there been a release in the last year?
```

**Expected behavior**

The `v0.4.0` API behavior of not erroring (c.f. above)

**Comment by matthewfeickert** [comment]
This took place in commits:
* e1e11f2866cc7fe0607905a67edddcae2c499534
* b657daae76a59dd8caa2a69293480bddabff7beb

However, nothing in the top level of the repository is pacakged unless it is [a default file (like `setup.py` and `setup.cfg`) or if it is included in `MANIFEST.in`](https://packaging.python.org/guides/using-manifest-in/#using-manifest-in).

(Aside, I would highly recommend adopting a `src` directory layout as demoed in the [PyPA Packaging Python Projects tutorial](https://packaging.python.org/tutorials/packaging-projects/). TL;DR: Using `src` makes it easier to be sure that you know what you're packaging and also makes it easier/safer to ensure that you're testing the installed version of your package and not the contents of your GitHub repository. I'm happy to discuss this on another Issue and provide examples if that is of any interest as well. :+1: )

**Comment by ryancheley** [comment]
@matthewfeickert thanks for the bug report. I've been able to replicate it locally and fix in version 0.5.1 which was just pushed out

I'll take a look at the link for using `src` and have created #15 for this 👍

**Comment by ryancheley** [comment]
Issue resolved with 5aa92fd

**Comment by matthewfeickert** [comment]
> I've been able to replicate it locally and fix in version 0.5.1 which was just pushed out

Great! Thanks. :)
